packages/api/src/schema/api-schema.yaml

```diff
@@ -2391,10 +2391,42 @@ components:
           type: number
           description: Total minutes of storage usage.
           example: 100
+    realtime-viewership-metric:
+      type: object
+      description: |
+        An individual metric about realtime viewership of a stream/asset.
+      required:
+        - viewCount
+        - errorRate
```

Comment:
Shouldn't the `playbackId` here be required, because it is the only filter to query results. Otherwise LGTM ✅

Reply:
If the query is done against the whole account (project id / user id), then we don't include `playbackId` in the response.